TheAlienThatEatsTheCarrot/Events/GameEvents/PlayerMoveEvent.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  PlayerMoveEvent.swift
+//  TheAlienThatEatsTheCarrot
+//
+//  Created by Justin Cheah Yun Fei on 27/3/24.
+//
+
+import Foundation
+
+struct PlayerMoveEvent: Event {
```

Comment:
how about calling it `PlayerControlActionEvent` or something similar? since player may have different actions like attack apart from move.

Reply:
Right will change that! Thanks!

Reply:
updated!